app/(routes)/organizers/page.tsx

```diff
@@ -12,37 +12,149 @@ const team = [
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: 'Features',
+    imageUrl:
+      "https://media.licdn.com/dms/image/D4E03AQHQ8rT5p0b8hQ/profile-displayphoto-shrink_200_200/0/1687527128902?e=1695254400&v=beta&t=2m2jtGGibeFC64K2HGRLp98QZYm8q7iWrooMPZSz69U"
+  },
+  {
+    name: <a href="https://www.linkedin.com/in/jingyinghuang/">Jingying Huang</a>,
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: 'Social Media',
+    imageUrl:
+      "https://media.licdn.com/dms/image/C4E03AQFuDEdSgOJPBw/profile-displayphoto-shrink_200_200/0/1571248783675?e=1695254400&v=beta&t=n2dFY1bI2U2iGFcItLaB3C8-4YcaCNQJ9X1ttYNoHJA"
+  },
+  {
+    name: <a href="https://www.linkedin.com/in/nicole-lyu/">Nicole Lyu</a>,
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: 'Social Media',
+    imageUrl:
+      "https://media.licdn.com/dms/image/C4E03AQG7_VeBySraNQ/profile-displayphoto-shrink_200_200/0/1661273766047?e=1695254400&v=beta&t=udWGSp3_7cOzvYqcelI1-yOPAom2MaO-20B591E9qeI"
+  },
+
+  {
+    name: <a href="https://www.linkedin.com/in/sergio-terrero-34045a11b/">Sergio Terrero</a>,
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: '',
+    imageUrl:
+      "https://media.licdn.com/dms/image/D4E03AQH2PwcKtDoGAg/profile-displayphoto-shrink_200_200/0/1670726619198?e=1695254400&v=beta&t=pbrUCiZcFNtqf6pavxPvcCiYmAezxWbYABuOCnTrymQ"
+  },
+  {
+    name: <a href="https://www.linkedin.com/in/shirly-spikes-725a413b/">Shirly Spikes</a>,
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: 'Features',
+    imageUrl:
+      "https://media.licdn.com/dms/image/C4D03AQF3qK7KUlfxHQ/profile-displayphoto-shrink_200_200/0/1609945840249?e=1695254400&v=beta&t=0WlFasb-0atWKFhlY4_XqyYP2W7CE1zAuE7z0WnK2h0"
+  },
+  {
+    name: <a href="https://www.linkedin.com/in/dutt-sai/">Sai Dutt</a>,
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: '',
+    imageUrl:
+      "https://media.discordapp.net/attachments/1130964544706056303/1133814348565975050/PXL_20230101_190254075_2.jpg?width=756&height=920"
+  },
+  {
+    name: <a href="https://www.linkedin.com/in/seancdaly/">Sean Daly</a>,
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: '',
+    imageUrl:
+      "https://media.licdn.com/dms/image/D5603AQEo8O0zmNabkA/profile-displayphoto-shrink_200_200/0/1689400196234?e=1695254400&v=beta&t=CIQJ7UX5cwpPA0mMQ7WyzsQ3TMKzbZ00CVNX59I--Z8"
+  },
+  {
+    name: <a href="https://www.linkedin.com/in/claudioolmedo-com/">Claudio Olmedo</a>,
+    role: "Volunteer",
+    // teamLead: 'Team Lead:',
+    team: '',
+    imageUrl:
+      "https://media.licdn.com/dms/image/D4E03AQE6q5fDJj5zcA/profile-displayphoto-shrink_200_200/0/1685118707659?e=1695254400&v=beta&t=MTVN3lH51OFLr27bpGGep401OW3RffbA8vNeANqZGZY"
+  },
+
+
+
 ];
 
 export default function Organizers() {
   return (
 
     <Wrapper title="Team" description="Meet our Organizers!">
```

Comment:
To match the Figma file, you should add two sections here. The top one has the text "Meet our Organizers!" and the second "Meet our Volunteers!".

Reply:
filtered the array to separate organizers and volunteers